### PR TITLE
chore: install standard-version before running bump

### PR DESCRIPTION
**Pull request by iliapolo**
This [PR](https://github.com/aws/aws-cdk/pull/11307) migrated our CLI usage of `standard-version` to be used in code as a library. But the library is not installed anywhere.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by rix0rrr — APPROVED** [review]

**Review by eladb — CHANGES_REQUESTED** [review]
Would it make more sense to simply run `yarn` from the root ?

**Comment by iliapolo** [comment]
> Would it make more sense to simply run `yarn` from the root ?

Debatable, the script used to be self-contained with `npx`, this simulates that.

Im going to ride on rico's approval and let it merge :)

**Comment by eladb** [comment]
It's not really the same thing because we are now installing the module locally so it can potentially "pollute" our node modules

**Comment by iliapolo** [comment]
> It's not really the same thing because we are now installing the module locally so it can potentially "pollute" our node modules

Fair enough, i'll change

File: scripts/bump-candidate.sh [hunk 1]
```diff
@@ -17,4 +17,6 @@
 # --------------------------------------------------------------------------------------------------
 set -euo pipefail
 scriptdir=$(cd $(dirname $0) && pwd)
+rootdir=${scriptdir}/..
+(cd ${rootdir} yarn standard-version)
 BUMP_CANDIDATE=true ${scriptdir}/bump.js ${1:-minor}
```
Comment (eladb) [inline scripts/bump-candidate.sh:22]:
```suggestion
BUMP_CANDIDATE=true ${rootdir}/bump.sh ${1:-minor}
```

File: scripts/bump-candidate.sh [hunk 2]
```diff
@@ -17,4 +17,6 @@
 # --------------------------------------------------------------------------------------------------
 set -euo pipefail
 scriptdir=$(cd $(dirname $0) && pwd)
+rootdir=${scriptdir}/..
+(cd ${rootdir} yarn)
 BUMP_CANDIDATE=true ${scriptdir}/bump.js ${1:-minor}
```
Comment (eladb) [inline scripts/bump-candidate.sh:22]:
```suggestion
BUMP_CANDIDATE=true ${rootdir}/bump.sh ${1:-minor}
```

File: bump.sh [hunk 3]
```diff
@@ -13,4 +13,6 @@
 #
 # --------------------------------------------------------------------------------------------------
 set -euo pipefail
-./scripts/bump.js ${1:-minor}
+scriptdir=$(cd $(dirname $0) && pwd)
+(cd ${scriptdir} yarn)
```
Comment (eladb) [inline bump.sh:17]:
```suggestion
cd ${scriptdir}
yarn
```

**Review by eladb — APPROVED** [review]

**Comment by eladb** [comment]
Sorry about this F up

**Pull request merged**